apps/vr-tests/package.json

```diff
@@ -35,7 +35,7 @@
     "@types/react-dom": "16.8.4",
     "@types/webpack-env": "1.13.9",
     "@uifabric/build": "^7.0.0",
-    "@uifabric/date-time": "^7.8.9",
+    "@uifabric/date-time": "^7.8.8",
```

Comment:
The changes in this file don't seem right

Reply:
This file has been reverted - there's no need to touch this file.